lib/Config.js

```diff
@@ -268,6 +268,19 @@ class Config {
                 this.vaultd.host = config.vaultd.host;
             }
         }
+        if (config.dataDaemon) {
+            this.dataDaemon = {};
+            assert.strictEqual(typeof config.dataDaemon.host, 'string',
+                               'bad config: data daemon host must be ' +
+                               'a string');
+            this.dataDaemon.host = config.dataDaemon.host;
+
+            assert(Number.isInteger(config.dataDaemon.port)
+                   && config.dataDaemon.port > 0,
+                   'bad config: dataDaemon port must be a positive ' +
+                   'integer');
```

Comment:
should we check to make sure it's not the same port as the s3 server and metadata server?